elfi/methods/methods.py

```diff
@@ -725,14 +725,15 @@ def __init__(self, model, target=None, outputs=None, batch_size=1, n_acq=150,
             Only needed if model is an ElfiModel
         target_model : GPyRegression, optional
         acquisition_method : Acquisition, optional
+            Method of acquiring evidence points. Defaults to LCBSC.
         bounds : list
             The region where to estimate the posterior for each parameter in
             model.parameters.
             `[(lower, upper), ... ]`
         initial_evidence : int, dict
             Number of initial evidence or a precomputed batch dict containing parameter 
             and discrepancy values
-        n_evidence : int
+        n_acq: int
```

Comment:
We also then wouldn't need `_n_initial_runs` since the only thing that matters is the `n_evidence`. I essence, all the evidence you gather will turn into `initial_evidence` for the future when you wish to continue where you are now.